src/org/intellij/erlang/psi/impl/ErlangPsiImplUtil.java

```diff
@@ -731,6 +731,12 @@ private static int calculateFunctionClauseArity(@NotNull ErlangFunctionClause cl
     return argumentDefinitionList.getArgumentDefinitionList().size();
   }
 
+  @Nullable
+  public static String getName(@NotNull ErlangAppParameter parameter) {
+    PsiElement atomName = parameter.getCurlyLeft().getNextSibling();
+    return atomName != null ? atomName.getText() : null;
```

Comment:
This is BAD. What if a next sibling is a comment? What if it's a whitespace? What if there's an error? for all of these cases you'll return some weird name instead of null.